.github/workflows/integration-test.yml

```diff
@@ -3,12 +3,16 @@ name: Run integration tests
 on:
   workflow_dispatch:
     inputs:
-      database:
-        description: 'Database - a new one will be created if not provided'
+      database1:
+        description: 'Database (v1) - a new one will be created if not provided'
         required: false
         default: ''
-      engine:
-        description: 'Engine - a new one will be created if not provided'
+      database2:
+        description: 'Database (v2) - a new one will be created if not provided'
+        required: false
+        default: ''
+      engine2:
+        description: 'Engine (v2) - a new one will be created if not provided'
```

Comment:
What about `engine1`?

Reply:
Somehow this parameter is absent in our definition in master branch, so I have not added it here too.

Reply:
Let's add it for completeness. Shouldn't be a big change, since you're changing the workflow anyway.